src/sha1_hash.cpp

```diff
@@ -95,7 +94,7 @@ namespace libtorrent
 			v |= v >> 8;
 			v |= v >> 16;
 
-			return ret + MultiplyDeBruijnBitPosition[
+			return ret + 31 - MultiplyDeBruijnBitPosition[
```

Comment:
hmm...do you think this is a bug that could be affecting arm/android binaries using RC_1_1?